### ISSUE TITLE
Not working in emacs 27

### ISSUE THREAD

**Issue by Ergus** [Lindydancer/e2ansi#9]
Hi: 
I just moved to emacs 27 and the e2ansy-cat  doesn't work at all. When I try it in the old emacs it works perfectly, but not in the newer (future release) of emacs.

Any suggestion?

**Comment by Lindydancer** [comment]
Hi!

I've been away from Emacs development for a while, so I haven't tracked
Emacs 27 lately.

I'll take a look at it as soon as I can find a suitable time slot.

    -- Anders

**Comment by Lindydancer** [comment]
Hi!

Just a quick status update. I've located the main problem, and fixed it.
However, in my tests I see various small problems that I would like to
address before I push a new version to GitHub.

The underlying problem is that batch frames in Emacs 27 say that they are
dark background in mono, which e2ansi adheres to.

Anyway, a quick workaround for you is to specify the command-line options
--background-mode=light and --color-class=color to e2ansi-cat to override
this.

    -- Anders

**Comment by Lindydancer** [comment]
Hi!

After some detective work I finally managed to figure out what was the problem!

In Emacs 27, frames in batch mode specify "mono" color class and "dark" background mode, and this package (well, actually the face-explorer package) adhered to this. In earlier versions, this was unspecified so the package assumed that color class "color" should be used with a light background. This was fixed by ignoring those two properties in for batch frames.

Also, I found a minor problem in the test code. In Emacs 27, themes must explicitly be enabled.

I've pushed new versions of e2ansi and face-explorer.

Thanks for reporting this!

    -- Anders
